recipe/meta.yaml

```diff
@@ -9,11 +9,13 @@ source:
   sha256: c1747eb8f26e9e753c4001ed951db2896edc1021b6d0f547a0bd2a27c30ada51
   patches:
     - g-ir-scanner-libpath.patch
+    - cmake_pkg_config_bat.patch  # [win]
+    - poppler_data_location.patch  # [win]
 
 build:
-  number: 1000
+  number: 1001
   detect_binary_files_with_prefix: true
-  skip: true  # [win or (cxx_compiler == 'toolchain_cxx')]
+  skip: true  # [(cxx_compiler == 'toolchain_cxx')]
```

Comment:
Hmm I wonder if this is preventing the smithy enabling Windows builds?

Reply:
I think that the problem is in the re-rendering webservice and not the `skip`. Let me try to re-render that locally.